[2025-02-09T03:54:01.472+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.process_user scheduled__2025-02-04T00:00:00+00:00 [queued]>
[2025-02-09T03:54:01.499+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.process_user scheduled__2025-02-04T00:00:00+00:00 [queued]>
[2025-02-09T03:54:01.499+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2025-02-09T03:54:01.500+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2025-02-09T03:54:01.500+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2025-02-09T03:54:01.755+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): process_user> on 2025-02-04 00:00:00+00:00
[2025-02-09T03:54:01.775+0000] {standard_task_runner.py:55} INFO - Started process 121 to run task
[2025-02-09T03:54:01.784+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'process_user', 'scheduled__2025-02-04T00:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp_afxa2pf']
[2025-02-09T03:54:01.790+0000] {standard_task_runner.py:83} INFO - Job 31: Subtask process_user
[2025-02-09T03:54:02.054+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.process_user scheduled__2025-02-04T00:00:00+00:00 [running]> on host fb1f760c5ca3
[2025-02-09T03:54:02.355+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=process_user
AIRFLOW_CTX_EXECUTION_DATE=2025-02-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-02-04T00:00:00+00:00
[2025-02-09T03:54:02.415+0000] {python.py:177} INFO - Done. Returned value was: None
[2025-02-09T03:54:02.447+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=process_user, execution_date=20250204T000000, start_date=20250209T035401, end_date=20250209T035402
[2025-02-09T03:54:02.607+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2025-02-09T03:54:02.686+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
